google-map-poly.html

```diff
@@ -0,0 +1,554 @@
+<!-- Copyright (c) 2015 Google Inc. All rights reserved. -->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-apis/google-maps-api.html">
+<link rel="import" href="google-map-point.html">
+
+<!--
+The `google-map-poly` element represents a series of connected line segments (aka a polyline) which
```

Comment:
This supports both polygons and polylines.  Calling it just polyline might confuse people familiar with the raw API and creating separate components would effectively duplicate most of the code so I thought it efficient to shorten the name and have a single implementation with a flag.